setup.py

```diff
@@ -1,8 +1,30 @@
 from setuptools import setup, find_packages
 from setuptools.command.install import install
 from setuptools.command.develop import develop
+from setuptools.command.build_py import build_py
 import sys
-import subprocess
+from os.path import join
+from os import walk
+
+STRIP_DIRS = ["src", "tests"]
+
+
+def strip_file(file_path):
+    from strip_hints import strip_file_to_string
+
+    if file_path.endswith('.py'):
+        stripped = strip_file_to_string(file_path, to_empty=True, only_assigns_and_defs=True)
+        print(file_path)
+        with open(file_path, 'w') as file_obj:
+            file_obj.write(stripped)
+
+
+def strip_all_hints(dirnames):
+    print('stripping type annotations from: {}'.format(dirnames))
```

Comment:
Is this a debugging print?

Reply:
Same as above, it's doing the same operation as the shell script.